raiden/network/transport/matrix.py

```diff
@@ -514,7 +537,15 @@ def _handle_message(self, room, event):
                     peer_address=pex(peer_address),
                     exception=ex,
                 )
-                return
+                return False
+            except InvalidProtocolMessage as ex:
+                self.log.warning(
```

Comment:
I think this should be a call to `error` ... unless we just want to warn and move on, then the UDP transport should also do this.

Reply:
yep we should just warn